apps/nextjs-website/src/components/molecules/WebinarQuestion/WebinarQuestionRow.tsx

```diff
@@ -0,0 +1,116 @@
+import { WebinarQuestion } from '@/lib/webinars/webinarQuestions';
+import { Box, IconButton, TableCell, TableRow, useTheme } from '@mui/material';
+import { CopyToClipboardButton } from '@pagopa/mui-italia';
+import DOMPurify from 'dompurify';
+import AutoFixHighIcon from '@mui/icons-material/AutoFixHigh';
+import VisibilityOffIcon from '@mui/icons-material/VisibilityOff';
+import Visibility from '@mui/icons-material/Visibility';
+import { useFormatter, useTranslations } from 'next-intl';
+import AutoFixOffIcon from '@mui/icons-material/AutoFixOff';
+
+type WebinarQuestionRowProps = {
+  question: WebinarQuestion;
+  userEmail: string;
+  onHighlight: (highlight: boolean) => Promise<void>;
+  onHide: (hidden: boolean) => Promise<void>;
+};
+
+export default function WebinarQuestionRow({
+  question,
+  userEmail,
+  onHide,
+  onHighlight,
+}: WebinarQuestionRowProps) {
+  const formatter = useFormatter();
+  const { palette } = useTheme();
+  const t = useTranslations('webinar.questionList');
+
+  const { hiddenBy, hiddenByFullName, highlightedBy, highlightedByFullName } =
+    question;
+
+  const isHidden = !!hiddenBy;
+  const isHighlighted = !!highlightedBy;
+
+  return (
+    <TableRow
+      hover
+      key={question.createdAt.toJSON()}
+      sx={{
+        '&:last-child td, &:last-child th': { border: 0 },
+        '&.MuiTableRow-hover:hover': {
+          backgroundColor: isHighlighted
+            ? palette.primary.dark
+            : palette.action.hover,
+        },
+        backgroundColor: isHighlighted ? palette.primary.light : '',
+        fontStyle: isHidden ? 'italic' : '',
+        position: 'relative',
+      }}
+    >
+      <TableCell sx={{ color: isHighlighted ? palette.common.white : '' }}>
+        {!isHidden
+          ? formatter.dateTime(question.createdAt, {
+              year: 'numeric',
+              month: 'numeric',
+              day: 'numeric',
+              hour: 'numeric',
+              minute: 'numeric',
+            })
+          : ''}
+      </TableCell>
+      <TableCell
+        width='70%'
+        sx={{ color: isHighlighted ? palette.common.white : '' }}
+      >
+        {!isHidden
+          ? question.question
+          : isHidden && hiddenBy === userEmail
+          ? t('hiddenByMe') + ' ( ' + question.question + ' )'
+          : t('hiddenBy') + hiddenByFullName}
```

Comment:
```suggestion
          ? `${t('hiddenByMe')} ("${question.question}")`
          : `${t('hiddenBy')} ${hiddenByFullName}`}
```
you can do this as well